.github/workflows/pull-requests.yml

```diff
@@ -155,14 +155,26 @@ jobs:
           webpackStatsFile: editor/lib/staging-stats.json
           key: ${{ secrets.RELATIVE_CI_KEY }}
           debug: false
+      - name: Delete node_modules
```

Comment:
Does this not mean we lose the cached content of the `node_modules` folders?

Reply:
I just pushed 40658c809 and 2a9ba9909 which caches `~/.pnpm` instead of the node_modules folders. Since the node_modules folders are just empty symlinks, there's no need caching them anymore.